src/v2/Apps/Fair/Components/FairEditorial/FairEditorial.tsx

```diff
@@ -22,7 +24,7 @@ export const FairEditorialFragmentContainer = createFragmentContainer(
   {
     fair: graphql`
       fragment FairEditorial_fair on Fair {
-        articles: articlesConnection(first: 5, sort: PUBLISHED_AT_DESC) {
+        articlesConnection(first: 6, sort: PUBLISHED_AT_DESC) {
```

Comment:
I wish I could just statically interpolate the const here but I don't believe there is a way to do that. And making this an argument proper is more confusing that the minor amount of duplication IMO